src/components/data_search/DatasetFilterList.jsx

```diff
@@ -8,13 +8,14 @@ import ListItemText from '@mui/material/ListItemText';
 import Divider from '@mui/material/Divider';
 import { Typography } from '@mui/material';
 import { Checkbox } from '@mui/material';
-import { flatten, uniq, compact, capitalize } from 'lodash';
+import { flatten, uniq, compact, capitalize, orderBy } from 'lodash';
 
 export const DatasetFilterList = (props) => {
   const { datasets, filters, filterHandler, isFiltered } = props;
 
   const accessManagementFilters = uniq(compact(datasets.map((dataset) => dataset.accessManagement)));
   const dataUseFilters = uniq(compact(flatten(datasets.map((dataset) => dataset.dataUse?.primary))).map((dataUse) => dataUse.code));
+  const dacFilters = orderBy(uniq(compact(datasets.map((dataset) => dataset?.dac?.dacName))), (dac) => dac.toLowerCase(), 'asc');
```

Comment:
Why does this use `dataset?` but other cases don't? It seems like it should be consistent here. (Also, it would be nice to move to tsx to have better type information here.)

Reply:
Not all datasets will have DACs, but all datasets will have a primary data use and an access management policy.